### PR TITLE
Add plugin docs

### PR DESCRIPTION
**Pull request by pditommaso**
Add plugins documentation

### CODE REVIEW

File: docs.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Make sure to have configured the XPACK license and the plugin as described 
+in the [README](README.md#configuration) document. 
+
+### Use of AWS EFS file system with Nextflow 
+
+AWS [EFS](https://aws.amazon.com/efs/) is a shared file-system based on the 
+NFS protocol provided by AWS. 
+
+To learn about EFS details and how to create EFS instance check the AWS documentation
+at [this link](XXXXXXX).
```
Comment (abhi18av) [inline docs.md]: Perhaps this is the link you intended to add? 
https://docs.aws.amazon.com/efs/latest/ug/creating-using-create-fs.html

File: docs.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Documentation 
+
+### AWS Batch pro execution 
+
+The `xpack-amzn` plugin implements an advanced version of AWS Batch executor 
+for Nextflow that allows using a shared file system in place of AWS S3 bucket 
+as the pipeline work directory and to ingest pipeline data.
```
Comment (abhi18av) [inline docs.md]:
```suggestion
as the pipeline work directory to ingest pipeline data. 
```

File: docs.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### Use of AWS EFS file system with Nextflow 
+
+AWS [EFS](https://aws.amazon.com/efs/) is a shared file-system based on the 
+NFS protocol provided by AWS. 
+
+To learn about EFS details and how to create EFS instance check the AWS documentation
+at [this link](XXXXXXX).
+
+Once you have create one or more file systen, to make accessible in your
```
Comment (abhi18av) [inline docs.md]:
```suggestion
Once you have createed one or more file systems, to make accessible in your 
```

File: docs.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+To use this file system as work directory in your Nextflow pipeline execution, 
+you will only need to specify a shared directory path in the Nextflow command-line 
+with the `-w` option, e.g. 
+
+```
+nextflow run <MY-PIPELINE> -w /mnt/some/shared/dir
+```
+
+To access shared file paths, other the pipeline work directory, requires to
```
Comment (abhi18av) [inline docs.md]:
```suggestion
To access shared file paths, other than the pipeline work directory, it is required to 
```

**Review by abhi18av — APPROVED** [review]
Minor suggestions, otherwise looks good to merge :)

File: docs.md (hunk 1 above)
Comment (pditommaso) [reply docs.md]: oops, yes!

File: docs.md (hunk 3 above)
Comment (pditommaso) [reply docs.md]: Typo also in the change 😄 I'll fix it

**Comment by pditommaso** [comment]
Cools thanks

**Pull request merged**